api/app/controllers/mno_enterprise/jpi/v1/admin/organizations_controller.rb

```diff
@@ -7,7 +7,8 @@ class Jpi::V1::Admin::OrganizationsController < Jpi::V1::Admin::BaseResourceCont
                        :geo_country_code, :geo_state_code, :geo_city,
                        :geo_tz, :geo_currency, :metadata, :industry, :size,
                        :financial_year_end_month, :credit_card,
-                       :financial_metrics, :created_at, :external_id]
+                       :financial_metrics, :created_at, :external_id, :orga_invites, :users,
```

Comment:
Should we get 2 separate sets of included fields for index and show as the fields added in the PR are related to associations?